reana_job_controller/kubernetes_job_manager.py

```diff
@@ -79,7 +82,7 @@ def execute(self):
                 'namespace': K8S_DEFAULT_NAMESPACE
             },
             'spec': {
-                'backoffLimit': current_app.config['MAX_JOB_RESTARTS'],
+                'backoffLimit': 0,
```

Comment:
Let's extract it to a variable so we document on it why we set it to 0 and what does it imply.